ciao-controller/legacy_api.go

```diff
@@ -270,43 +281,43 @@ func listServerDetailsFlavors(c *controller, w http.ResponseWriter, r *http.Requ
 
 func legacyComputeRoutes(ctl *controller, r *mux.Router) *mux.Router {
 	r.Handle("/v2.1/{tenant}/servers/action",
-		legacyAPIHandler{ctl, tenantServersAction}).Methods("POST")
+		legacyAPIHandler{ctl, tenantServersAction, false}).Methods("POST")
 
 	r.Handle("/v2.1/flavors/{flavor}/servers/detail",
-		legacyAPIHandler{ctl, listServerDetailsFlavors}).Methods("GET")
+		legacyAPIHandler{ctl, listServerDetailsFlavors, true}).Methods("GET")
 
 	r.Handle("/v2.1/{tenant}/resources",
-		legacyAPIHandler{ctl, listTenantResources}).Methods("GET")
+		legacyAPIHandler{ctl, listTenantResources, false}).Methods("GET")
 
 	r.Handle("/v2.1/{tenant}/quotas",
-		legacyAPIHandler{ctl, listTenantQuotas}).Methods("GET")
+		legacyAPIHandler{ctl, listTenantQuotas, false}).Methods("GET")
 
 	r.Handle("/v2.1/tenants",
-		legacyAPIHandler{ctl, legacyListTenants}).Methods("GET")
+		legacyAPIHandler{ctl, legacyListTenants, true}).Methods("GET")
```

Comment:
I wonder if this one shouldn't be admin only? I know BAT tests rely on user ciao-cli tenant list working.